src/main/java/com/atlassian/db/replica/internal/SQLOpeartionType.java

```diff
@@ -0,0 +1,50 @@
+package com.atlassian.db.replica.internal;
+
+public class SQLOpeartionType {
+
+    public static enum SQLType {
+        WRITE, READ_ONLY, NOT_KNOWN
+    }
+
+    private final SqlFunction sqlFunction;
+
+    public SQLOpeartionType(SqlFunction sqlFunction) {
+        this.sqlFunction = sqlFunction;
+    }
+
+    /**
+     * Arbitrially judge based on sql param whether sql is WRITE
```

Comment:
Typo in "Arbitrially".
Why arbitrarily though?

Reply:
I meant finally, without discussion, ultimately would be a better word here, sorry :)